www/includes/easyparliament/templates/html/divisions/vote.php

```diff
@@ -11,146 +10,101 @@
-                    <div class="debate-navigation__previous-debate">
-                        <a href="<?= $nextprev['prev']['url'] ?>" rel="prev">&laquo; <?= $nextprev['prev']['body'] ?></a>
-                    </div>
-                    <?php } ?>
+                <div class="debate-speech__division">
 
-                    <?php if (isset($nextprev['up'])) { ?>
-                    <div class="debate-navigation__all-debates">
-                        <a href="<?= $nextprev['up']['url'] ?>" rel="up"><?= $nextprev['up']['body'] ?></a>
-                    </div>
-                    <?php } ?>
+                    <h2 class="debate-speech__division__header">
+                        <img src="/images/bell.png">
+                        <small class="debate-speech__division__number">Division number <?= $division['number'] ?></small>
+                        <strong class="debate-speech__division__title"><?= $division['division_title'] ?></strong>
+                    </h2>
 
-                    <?php if (isset($nextprev['next'])) { ?>
-                    <div class="debate-navigation__next-debate">
-                        <a href="<?= $nextprev['next']['url'] ?>" rel="next"><?= $nextprev['next']['body'] ?> &raquo;</a>
+                  <?php if (isset($member)) { ?>
+                    <div class="debate-speech__division__your-mp">
+                      <?php if ( ! $main_vote_mp) { /* $main_vote_mp is true if an MP has been requested via the URL */ ?>
+                        <div class="your-mp__header">
+                          <h3>How your MP voted</h3>
+                          <p>
+                              Based on postcode <strong><?= $data['mp_data']['postcode'] ?></strong>
+                              <a href="<?= $data['mp_data']['change_url'] ?>">(Change postcode)</a>
+                          </p>
+                        </div>
+                      <?php } ?>
+                        <a href="<?= $member->url() ?>" class="your-mp__content">
+                          <?php if ( isset($mp_vote) ) { ?>
+                            <span class="your-mp__vote your-mp__vote--<?= $mp_vote['vote'] ?>"><?php
+                              switch ($mp_vote['vote']) {
+                                  case 'aye':
+                                      echo 'Aye';
+                                      break;
+                                  case 'no':
+                                      echo 'No';
+                                      break;
+                                  case 'absent':
+                                      echo 'Absent';
+                                      break;
+                                  case 'both':
+                                      echo 'Abstain';
+                                      break;
+                                  case 'tellaye':
+                                  case 'tellno':
+                                      echo 'Teller';
```

Comment:
We seem to have lost the information as to whether they were teller for the ayes or the noes.